perfmetrics/scripts/vm_metrics/vm_metrics.py

```diff
@@ -300,16 +365,20 @@ def fetch_metrics(self, start_time_sec, end_time_sec, instance, period, test_typ
       start_time_sec (int): Epoch seconds
       end_time_sec (int): Epoch seconds
       instance (str): VM instance
-      period (float): Period over which the values are taken 
+      period (float): Period over which the values are taken
       test_type(str): The type of load test for which metrics are taken
 
     Returns:
+      list[[period end time, interval end time,CPU_UTI_PEAK, CPU_UTI_MEAN,
+      REC_BYTES_PEAK, REC_BYTES_MEAN,SENT_BYTES_PEAK,SENT_BYTES_MEAN,OPS_ERROR_COUNT,
+      MEMORY_USAGE_PEAK,MEMORY_USAGE_MEAN,LOAD_AVG_OS_THREADS_MEAN]] in case of rename
+
       list[[period end time, interval end time, CPU_UTI_PEAK, CPU_UTI_MEAN,
       REC_BYTES_PEAK, REC_BYTES_MEAN, READ_BYTES_COUNT, OPS_ERROR_COUNT,
-      OPS_MEAN_LATENCY]]
+      OPS_MEAN_LATENCY]] otherwise
```

Comment:
otherwise?

Reply:
When test type is rename, only then we are returning the metrics mentioned in the RENAME_METRICS_LIST, but all any other cases ,by default , we are returning this metrics mentioned in METRICS_LIST